src/hbbft.erl

```diff
@@ -136,7 +137,9 @@ start_on_demand(Data = #hbbft_data{buf=Buf, n=N, secret_key=SK, batch_size=Batch
     %% time to kick off a round
     {NewACSState, {send, ACSResponse}} = hbbft_acs:input(Data#hbbft_data.acs, EncX),
     %% add this to acs set in data and send out the ACS response(s)
```

Comment:
Can we update this comment please? Something along the lines of "store our own stamps and decrypted shares since our proposal got included in the acs result"

Reply:
Done